WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCCrypto/VCCrypto/Algorithms/P256.swift

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import Foundation
+import CryptoKit
+
+enum P256Error: Error, Equatable
+{
+    case InvalidKeyMaterialInJWK
+    case JWKContainsInvalidKeyType(String)
+    case JWKContainsInvalidCurveAlgorithm(String?)
+    case MissingKeyMaterialInJWK
+    case NotImplemented
+}
+
+/// Algorithm that verifies a message using the P256 algorithm.
+struct P256: Signing {
```

Comment:
Naming in general for this file since its referenced as `WalletLibrary.<struct>` may be better as `ES256`.

Reply:
Makes sense to me!